database/queries.ts

```diff
@@ -134,6 +134,43 @@ function calculateStreak(entryTimes: number[], currentTime :number, waitTime :nu
   return streak;
 }
 
+export const readInventory = async () => {
+  const db = await dbPromise;
+  const rows : {InventoryOfItems:string}[]= await db.getAllAsync('SELECT InventoryOfItems FROM App');
+  console.log(rows);
+
+  const inventoryString = rows[0].InventoryOfItems;
+  const inventory = JSON.parse(inventoryString);
+
+  return(inventory);
+}
+
+export const updateInventory = async (inventory: ItemAndCount[]) => {
+  const db = await dbPromise;
+  const result = await db.runAsync('UPDATE App SET InventoryOfItems = ?', JSON.stringify(inventory));
+  console.log(`Updated ${result.changes} row(s)`);
+}
```

Comment:
When we write async functions, we generally can just return `db.runAsync` or whatever final async function. This lets callers access to the last job (and allows cancellation, etc.)

I would write these functions like `fetch`, allowing us to chain `then`, etc. in the callers